typescript/teamMembers.d.ts

```diff
@@ -23,6 +23,7 @@ interface RolePrivileges {
   FORM_SUBMISSION_ATTACHMENTS?: 'MANAGER'
   FORM_SUBMISSION_PREFILL?: 'MANAGER'
   WEBHOOK_SUBSCRIPTIONS?: 'MANAGER' | 'READONLY'
+  AI?: 'MANAGER' | 'READONLY'
```

Comment:
I expect we can let anyone read this data. Will need to check with Blake but if so, we dont need the "readonly" value. We can just check if they have team member record to allow them read access.